_includes/core/value_props.html

```diff
@@ -0,0 +1,63 @@
+<div id="vp-container">
+  <div id="initial-vps">
+    <div class="row value-props">
+      <div class="col-lg-4 text-center animated fadeInLeft">
+        <span class="fa fa-cubes fa-3x"></span>
+        <h2 class="value-prop-heading">Modular</h2>
+        <p>.NET Core is a set of runtime, library and compiler components. Microsoft uses these components in various configurations for device and cloud workloads. You can do the same for your app or service.</p>
```

Comment:
Would be great if we can talk about more companies than just Microsoft. Maybe not possible yet for .NET Core?